.github/workflows/deploy-dry-mainnet-events.yml

```
@@ -0,0 +1,41 @@
+name: Dry-run Deploy Widgets to Mainnet
+on:
+  pull_request:
+  push:
+    branches-ignore:
+      - main
+jobs:
+  deploy-widgets:
+    runs-on: ubuntu-latest
+    name: Diff from PR
```

```suggestion
    name: Deploy ( or diff from PR )
```
Just for consistency in all three workflows